scripts/sanity/runtime-rbac/base-test.js

```
@@ -0,0 +1,1285 @@
+        let encodedData;
+        
+        switch (actionType) {
+            case this.RoleConfigActionType.CREATE_ROLE:
+                // (string roleName, uint256 maxWallets, FunctionPermission[] functionPermissions)
+                // FunctionPermission is tuple(bytes4,uint16)
+                const functionPermsArray = data.functionPermissions.map(fp => [
+                    fp.functionSelector,
+                    fp.grantedActionsBitmap
+                ]);
+                encodedData = this.web3.eth.abi.encodeParameters(
+                    ['string', 'uint256', 'tuple(bytes4,uint16)[]'],
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)